### DISCUSSION TITLE
Unable to verify your status as a participant in the GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by gianluca-cacciarini** [community/community#106619] [GitHub Education]
### Hi everyone,

Hi, I'm experiencing a problem when trying to register for the MongoDB associate developer certificate.
I have already been approved to the GitHub Student Developer Pack.
The error message is as follows:

_Welcome and thanks for logging in. Unfortunately we are unable to verify your status as a participant in the GitHub Student Developer Pack. Please visit the GitHub Student Pack home for additional information and how to register._

Thanks in advance

**Comment by samus-aran** [comment]
Hi @gianluca-cacciarini 👋 

We can confirm that you currently have a verified GitHub Education student coupon 👍 If you are having issues with partners verifying you please raise a ticket with [ Education support  ](https://support.github.com/contact/education)using the dropdown **"I am a verified student having trouble redeeming my Global Campus offers"**.
